docs/section5/Rule5-17.md

```diff
@@ -0,0 +1,36 @@
+
+# Envelope - Rule 5-17  
+
+**Rule ID:** 5-17  
+**Rule Description:** Opaque surfaces that are not regulated (not part of opaque building envelope) must be modeled the same in the baseline as in the proposed design.  
+**Appendix G Section:** Section G3.1-5 Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: ```scc_dictionary_b = get_surface_conditioning_category(B_RMR)```  
+
+- For each building segment in the Proposed model: ```for building_segment_b in B_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```for thermal_block_b in building_segment_b.thermal_blocks:```  
+
+    - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
+
+      - For each surface in zone: ```for surface_b in zone_b.surfaces:```  
+
+        - Check if surface is unregulated: ```if ( scc_dictionary_b[surface_b.id] == UNREGULATED ):```  
+
+          - Get matching surface from P_RMR: ```surface_p = match_data_element(P_RMR, surfaces, surface_b.id)```  
+
+            **Rule Assertion:**  
+
+            Case 1: Surface construction in B_RMR matches P_RMR: ```if surface_b.construction == surface_p.construction: PASS```  
+
```

Comment:
Construction data object has multiple attributes. I think we can just check equivalency of U/C/F factors, surface_construction_input_option, and perhaps has_radiant_heating and has_radiant_cooling. @supriyagoel and @JasonGlazer - do you agree, or do you think we need to do a deep check covering all attributes? We don't need "Notes" field to match for example.

Reply:
I thought we had discussed some general approach for ensuring that everything not modified by rules was the same.

Reply:
I remember that also but I don't know what was decided. I think in order to minimize inconsequential RCT flags we may want to limit the sameness check to just the RMR attributes that are mandatory for App G. @supriyagoel should we check with James?

Reply:
@mkarpman I agree we only need to check u,f,c.
i dont think we need to check has radiant cooling/has radiant heating.
I dont remember a discussion on this topic but adding @mcneillj to confirm

Reply:
@supriyagoel I wonder why you think we shouldn't check "has radiant heating/cooling". I don't have strong feelings, but I think we should come up with logic for what to check. Obviously, we want the "regulated" inputs to align (U/C/F factors and modeling method for surfaces, LPDs for lighting, etc.) Should we skip all other data points that may be included in the RMR?

Reply:
As far as I understand, the RDS is currently checking for the full alignement of the surface.construction object between P-RMR and U-RMR for unregulated surfaces. Didn't we decide that we only check the regulated aspects, i.e. U/C/F-factors and optical properties for the roofs? I don't have strong feelings either way.